app/controllers/api/v2/override_values_controller.rb

```diff
@@ -104,6 +105,11 @@ def rename_use_puppet_default
         params[:override_value][:omit] = params[:override_value].delete(:use_puppet_default)
         Foreman::Deprecation.api_deprecation_warning('"use_puppet_default" was renamed to "omit"')
       end
+
+      def validate_value
```

Comment:
A comment explaining why we're validating things in the API vs models would be helpful for other devs